tools/prepare_python_system_tests.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import json
+import os
+import subprocess
+import tempfile
+import time
+import urllib.request
+import zipfile
+
+parser = argparse.ArgumentParser(description='Downloads the latest release nimi-python and runs system tests on the specified driver.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('-d', '--driver', required=True, type=str, help='Driver Name.')
```

Comment:
I feel weird calling this driver name. It's really the Python module "nidmm" not the driver name "NI-DMM".

The description above also calls it "driver". Consider: 
* "on the specified Python module".
* --driver -> --module with help "Python module name e.g. `nidmm`, `niscope`, etc.".

Reply:
Changed the help string to `'Python package name.'`